config/locales/views/pages/secondary-senior-leaders/en.yml

```diff
@@ -2,107 +2,54 @@ en:
               url: "/curriculum"
-              event-label: "Curriculum resources"
-      student-help:
-        image:
-          alt: 'A teacher teaching'
-        logo:
-          alt: 'Isaac logo'
-        title: "How we can help you and your students"
-        text: "Isaac Computer Science is our free online learning programme for A level and GCSE computer science students and teachers, providing learning materials that cover the AQA, Edexcel, Eduqas, OCR and WJEC curriculums."
-        button:
-          title: "Discover Isaac Computer Science"
-          url: "https://isaaccomputerscience.org/"
-          event-label: "Isaac CS"
-      partners:
-        title: "Explore our partners"
-        text:
-          html: "Building young peoples’ skills is an essential part of our commitment to developing a world-leading computing education. Our %{educational_partners_link} offer complementary programmes to help your school or college provide inspirational computing activities and extra-curricular experiences for your students."
-          educational-partners-link:
-            title: "educational partners"
-            url: "/contributing-partners"
-            event-label: "Partners"
-        cards:
-          stem:
-            title: "STEM Ambassadors"
-            text: "Relatable professionals who support learning, raise aspirations and illuminate careers - including those in computing and the digital sector - face to face or virtually, free of charge."
-            link:
-              title: "Engage with STEM Ambassadors"
-              url: "https://www.stem.org.uk/stem-ambassadors"
-              event-label: "STEM Ambassadors"
-          code-club:
-            title: "Code Club"
-            text: "An extracurricular club for young people aged 9 to 13. Clubs are organised by teachers and volunteers, and supported by free projects and resources from Code Club."
+              event-label: "curriculum"
+          isaac:
+            title: "Isaac Computer Science"
+            text:
+              html: "The free online textbook for GCSE and A level Computer Science teachers and students with resources covering the full English curriculum and all exam specifications.<br /><br />Isaac Computer Science is great for supporting remote teaching, homework and revision sessions, and helps you to quickly identify areas where students can improve."
             link:
-              title: "Start or join a club"
-              url: "https://codeclub.org/en/"
-              event-label: "Code Club"
-          bebras:
-            title: "Bebras UK"
-            text: "The Bebras Computing Challenge introduces computational thinking to students. It is organized in over 50 countries and designed to get students all over the world excited about computing."
+              title: "Discover Isaac Computer Science"
+              url: "/issac_computer_science"
+              event-label: "issac computer science"
```

Comment:
This URL looks like it should be `/isaac-computer-science` instead. Also beware the misspelling of "issac" here.